scripts/geth-allowed-packages.txt

```diff
@@ -17,4 +17,6 @@
 "github.com/ethereum/go-ethereum/ethdb/pebble"
 "github.com/ethereum/go-ethereum/event"
 "github.com/ethereum/go-ethereum/log"
+"github.com/ethereum/go-ethereum/metrics"
+"github.com/ethereum/go-ethereum/metrics.(*meterArbiter).tick"
```

Comment:
does that work? not sure what does this even mean

Reply:
I removed it, let's see if CI complains 👀  But yeah this definitely looks sketchy.